### PR TITLE
Make Puppetboard work with the PuppetDB V4 api

### PR DESCRIPTION
**Pull request by raphink**
This PR adapts puppetboard to work with https://github.com/puppet-community/pypuppetdb/pull/58

All the features seemed to work fine with that.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,5 +5,5 @@ MarkupSafe==0.19
 WTForms==1.0.5
 Werkzeug==0.9.4
 itsdangerous==0.23
-pypuppetdb==0.1.1
+git+https://github.com/raphink/pypuppetdb@v4-api#egg=pypuppetdb-0.1.0
```
Comment (raphink) [inline requirements.txt]: This will obviously have to be adapted when pypuppetdb is released.

File: setup.py [hunk 2]
```diff
@@ -32,7 +32,9 @@
         "Flask >= 0.10.1",
         "Flask-WTF >= 0.9.4, <= 0.9.5",
         "WTForms < 2.0",
-        "pypuppetdb >= 0.1.0",
+        ],
+    dependency_links=[
+        "git+https://github.com/corey-hammerton/pypuppetdb@v4-api#egg=pypuppetdb",
```
Comment (raphink) [inline setup.py]: Same here.

**Comment by raphink** [comment]
@corey-hammerton Whenever pypuppetdb is released, I can update this to fix setup.py

**Comment by mcanevet** [comment]
@raphink could you please update this? I'm sure you have a lot of free time these days :-)

**Comment by raphink** [comment]
@mcanevet what I don't have is a computer with my SSH key on it :wink: but I can give you rights on my repository so you can fix it.

**Comment by raphink** [comment]
@mcanevet I gave you rights on my repository so you can now amend my branches for all PRs. As far as this PR is concernced, only setup.py and requirements.txt need to be modified iirc.

**Comment by mcanevet** [comment]
@corey-hammerton you can merge this now and probably release 0.1.0

**Comment by raphink** [comment]
Thanks @mcanevet

**Pull request merged**